docs/articles/documentation/test-api/assertions/index.md

```diff
@@ -76,6 +76,7 @@ To stabilize such tests, you need to add a timeout that will guarantee that the
 Adding such timeouts can increase the test running time because of extra waiting.
 
 If the TestCafe assertion receives a [Selector's DOM node state property](../selecting-page-elements/selectors.md#define-assertion-actual-value)
+or a [client function](../obtaining-data-from-the-client.md)
 as an actual value, TestCafe uses the smart assertion query mechanism:
 if an assertion did not pass, the test does not fail immediately. The assertion retries to pass multiple times and
 each time it requests the actual property value. The test fails if the assertion could not complete successfully
```

Comment:
And one other place https://devexpress.github.io/testcafe/documentation/test-api/assertions/#optionstimeout
`The amount of time, in milliseconds, allowed for an assertion to pass before the test fails if a selector property was used in assertion.`
add please about client function too